molecule/default/molecule.yml

```diff
@@ -27,29 +27,25 @@ driver:
 
 platforms:
   - name: ${name:-instance}
-    image: ${image:-centos:7}
-    command: /usr/sbin/init
+    image: "geerlingguy/docker-${image:-centos7}-ansible:latest"
+    command: ${MOLECULE_DOCKER_COMMAND:-""}
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:ro
     tmpfs:
       - /run
       - /tmp
-    # privileged: true
-    groups:
-      - ${pygroup:-python}
+    privileged: true
+    pre_build_image: true
```

Comment:
Changed a bit this file, I followed this [article](https://www.jeffgeerling.com/blog/2018/testing-your-ansible-roles-molecule), as you can see I deleted the `dockerfile.j2` to adopt these pre-built image with **ansible** support and systemd and are working very well with zero effort.